EFDataAccessLibrary/Features/Estimates/Estimate.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace EFDataAccessLibrary.Features.Estimates
+{
+    public sealed class Estimate
+    {
+        public int Id { get; set; }
+        public int Value { get; set; }
+        public string Name { get; set; }
+        public string Type { get; set; }
+        // TODO: change to enum
+    }
+
+    public sealed class EstimateDto
+    {
+        public int Id { get; set; }

```

Comment:
Please create constructor for that DTO. Now you can create DTO object e.g with ID null. You should create constructor with all properties. DTO should be readonly please remove all set. If you have only getter you can set values only in constructor